sale_commission/tests/test_sale_commission.py

```diff
@@ -108,14 +108,14 @@ def test_sale_commission_gross_amount_payment(self):
             1
         )
 
-    def test_sale_commission_gross_amount_payment_annual(self):
+    def _test_sale_commission_gross_amount_payment_annual(self):
         self.check_full(
             self.agent_annual,
             self.commission_section_paid,
             12
         )
 
-    def test_sale_commission_gross_amount_payment_semi(self):
+    def _test_sale_commission_gross_amount_payment_semi(self):
```

Comment:
Why is this needed?

Reply:
Nothing. I forgot to restore it. Restoring it...

Reply:
Done